benchmark/misc/startup-cli-version.js

```diff
@@ -11,10 +11,10 @@ const path = require('path');
 // indispensible part of the CLI.
 const bench = common.createBenchmark(main, {
   cli: [
-    'tools/node_modules/eslint/bin/eslint.js',
```

Comment:
Why not just check the existence of them here when creating the benchmark? If they don't exist, just don't push them into the array. The benchmark runner only takes the array at runtime.

Reply:
This way the array is static, otherwise it becomes dynamic which somehow makes it more complicated for little benefit. I think that just changing this order is enough for the tests that are run.

Reply:
I've moved the check to filter the list, as suggested. It duplicates the `path.resolve()` call but there seems to be no way of avoiding this without putting resolved path into the test output.